CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+v3.3.0.0
```

Comment:
Could you revert this part please?

Reply:
Definitely! Wasn't sure if this had to be part of the PR anyway. 👍